set.mm

```diff
@@ -138351,6 +138364,14 @@ the existence of its supremum (see ~ suprcl ).  (Contributed by Paul
   elfzelz $p |- ( K e. ( M ... N ) -> K e. ZZ ) $=
     ( cfz co wcel cuz cfv cz elfzuz eluzelz syl ) ABCDEFABGHFAIFABCJBAKL $.
 
+  ${
+    elfzelzd.1 $e |- ( ph -> K e. ( M ... N ) ) $.
+    $( A member of a finite set of sequential integer is an integer.
```

Comment:
integers

Reply:
Ah I just copy pasted, then that needs fixing too https://us.metamath.org/mpeuni/elfzelz.html